### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by igor-bratko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# training
+Test readme for commit
```
Comment (annigilator1) [inline README.md:2]: Need change line

Comment (annigilator1) [inline README.md:2]: Comment

**Review by annigilator1 — COMMENTED** [review]
Please, fix followed issues

File: README.md (hunk 1 above)
Comment (igor-bratko) [reply README.md:2]: Done

Comment (igor-bratko) [reply README.md:2]: Fixed

**Review by annigilator1 — APPROVED** [review]

**Pull request merged**